capstone/src/main/java/com/google/sps/servlets/CheckStatus.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/status")
+public class CheckStatus extends HttpServlet {
```

Comment:
Should this be called `CheckUserStatus` to be more explicit about what we are checking the status for?